src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item", null)
+                .ResolveAsync( async x =>
```

Comment:
because your resolving a service that likely uses sql further down the stack please use the `ResolveLockedAsync` extension, this will stop thread safety issues with YesSQL sessions when not using dataloaders

Reply:
fixed